src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -73,7 +73,7 @@ internal sealed class LocalFunctionSymbol : MethodSymbol
             }
 
             _binder = binder;
-            _refKind = (syntax.ReturnType.Kind() == SyntaxKind.RefType) ? RefKind.Ref : RefKind.None;
+            _syntax.ReturnType.SkipRef(out _refKind);
```

Comment:
Consider adding a helper to make this idiom clearer. "Skipping" the ref to get the RefKind is not very obvious.